plugins/flatpak/gs-flatpak.c

```diff
@@ -4062,9 +4066,17 @@ gs_flatpak_update_app (GsFlatpak *self,
 		}
 	}
 
-	if (!gs_flatpak_has_space_to_update (self, app, list)) {
-		g_debug ("Skipping automatic update for %s: not enough space on disk",
+	is_auto_update = gs_utils_app_is_auto_updating (app);
+	if (!gs_flatpak_has_space_to_update (self, app, list, is_auto_update)) {
+		g_debug ("Skipping %s for %s: not enough space on disk",
+			 (is_auto_update ? "automatic update" : "update"),
 			 gs_app_get_unique_id (app));
+		gs_app_set_state_recover (app);
+		if (!is_auto_update)
```

Comment:
Add a comment about why we're not reporting an error.
`/* Do not return an error because we don't want to see a banner showing up out of the blue for possibly multiple update failures */`